lsm-perf.py

```diff
@@ -0,0 +1,180 @@
+#!/usr/bin/env
+
+import argparse
+import time
+import plumbum
+import sys
+import statistics
+from plumbum import local, SshMachine  # TODO: requirements
+
+
+ROUNDS = 3
+REPETITIONS_PER_ROUND = 50
+WARMUP_RUNS = 5
+ON_VM_WORKLOAD_PATH = '~/lsm-perf-workload'
+QEMU_EXIT_CMD = b'\x01cq\n'  # -> ctrl+a c q
+
+
+def main(args):
+    try:
+        init_output_file(args.out)
+        for round in range(ROUNDS):
+            print('Starting round %d' % round)
+            for kernel in args.kernels:
+                results = evaluating_kernel(
+                    kernel_path=kernel.name,
+                    img_path=args.image.name,
+                    workload_path=args.workload.name,
+                    keyfile=args.key.name
+                )
+                write_results_to_file(args.out, kernel.name, round, results)
+    except KeyboardInterrupt:
+        print('\nExit prematurely')
+    finally:
+        args.out.close()
+    return 0
+
+
+def evaluating_kernel(kernel_path, img_path, workload_path, keyfile):
```

Comment:
I'd call this `evaluate_kernel` rather than "evaluating"

Reply:
Sounds good